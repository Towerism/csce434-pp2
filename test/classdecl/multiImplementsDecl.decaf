class Asdf implements Fdsa, Qwer, Rewq {
      int asdf;
}