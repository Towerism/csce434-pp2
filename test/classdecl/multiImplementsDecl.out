
   Program: 
  1   ClassDecl: 
  1      Identifier: Asdf
  1      (implements) NamedType: 
  1         Identifier: Fdsa
  1      (implements) NamedType: 
  1         Identifier: Qwer
  1      (implements) NamedType: 
  1         Identifier: Rewq
  2      VarDecl: 
            Type: int
  2         Identifier: asdf
